README.md

```diff
@@ -82,6 +82,41 @@ s := sanitizer.New(sanitizer.OptionDateFormat{
 })
 ```
 
+### Custom Sanitizers
+
+Use this option to register a custom sanitizer function. The sanitizer function is responsible for determining if the field's type is supported for that sanitizer.
+
+The `Name` field tells us what tag name corresponds to the sanitizer.
+
+The `Sanitizer` field tells us which sanitizer function to call when this tag is used. All sanitizers must have this signature: `func(s Sanitizer, structValue reflect.Value, idx int) error`.
+
+```go
+func exclaim(s Sanitizer, structValue reflect.Value, idx int) error {
```

Comment:
Can you add a small comment describing what this sanitizer does?

Reply:
Done